packages/web-components/@sfx/autocomplete/stories/index.ts

```diff
@@ -0,0 +1,78 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import '../src/index.ts';
+
+const tbd = {}
+
+const results = [
+  {
+    title: 'Brands',
+    items: [
+      {
+        label: 'New Balance',
+        ...tbd
+      }, {
+        label: 'Bauer',
+        ...tbd
+      }
+    ]
+  }, {
+    title: 'default',
+    items: [
+      {
+        label: 'Golf Ball',
+        ...tbd
+      }, {
+        label: 'Basketball',
+        ...tbd
+      }
+    ]
+  }
+];
+
+const optionalTitle = 'Autocomplete Results';
+
+  /**
+   * --- setup for testing event listeners ---
+   * Because Storybook is contained in an iFrame, for testing purposes, 
+   * we are unable to dispatch events directly from the console.
+   * 
+   * As an alternative, we use a timeout to dispatch the event below.
+   * 
+   * After the timeout, we should see the original data being replaced by the data contained in the custom event below.
+   */
+const autocompleteDataReceivedEvent = new CustomEvent('sfx::autocomplete_received_results', { detail: [{"title":"Brands","items":[{"label":"Cats"},{"label":"Dogs"}]},{"title":"default","items":[{"label":"Cars"},{"label":"Bikes"}]}],
+bubbles: true });
+
+setTimeout(() => {
+  window.dispatchEvent(autocompleteDataReceivedEvent)
+}, 3000)
+  /**
+  * --- end of setup for testing event listeners ---
+  */
+
+  /**
+   * --- Slots Testing ---
+   * The final story demonstrates that content slotted in (in this case, the <div> contained within the autocomplete custom element)
+   * appears within the light-dom as desired. 
+   * 
+   */
+
+storiesOf('Components|Autocomplete', module)
+  .addDecorator(withKnobs)
+    .add('Default', () => `
+      <sfx-autocomplete results="${text('Autocomplete Results', JSON.stringify(results))}"></sfx-autocomplete>
```

Comment:
No action needed here as this discussion is out of scope for this ticket, but I'm still not convinced that we _must_ pass these in as attributes because open-wc has a [library](https://open-wc.org/demoing/#manual) that lets one use lit-html for story templates (I assume this library can be used without the rest of open-wc).